### PR TITLE
feat(dashboard): Pause workflow from the context actions on workflow list

### PR DESCRIPTION
**Pull request by BiswaViraj**
### Screenshots

https://github.com/user-attachments/assets/c03b16a3-34e1-4868-b2e2-7d41d1cde7d1

<details>
<summary><strong>Expand for optional sections</strong></summary>

### Special notes for your reviewer

</details>

### CODE REVIEW

File: apps/api/src/app/workflows-v2/mappers/notification-template-mapper.ts [hunk 1]
```diff
@@ -63,10 +63,22 @@ function toMinifiedWorkflowDto(template: NotificationTemplateEntity): WorkflowLi
...
+    return WorkflowStatusEnum.ERROR;
+  }
+
+  if (template.active) {
+    return WorkflowStatusEnum.ACTIVE;
+  }
+
+  return WorkflowStatusEnum.INACTIVE;
+}
+
```
Comment (rifont) [inline apps/api/src/app/workflows-v2/mappers/notification-template-mapper.ts]: I think this state should come from the backend, we shouldn't be computing the status on the frontend.

Comment (BiswaViraj) [reply apps/api/src/app/workflows-v2/mappers/notification-template-mapper.ts]: @rifont this change is in the backend API 😅

Comment (djabarovgeorge) [reply apps/api/src/app/workflows-v2/mappers/notification-template-mapper.ts]: true it is computed once we know that issue,
PostProcessWorkflowUpdate.overloadStatusOnWorkflow

**Review by rifont — APPROVED** [review]

File: apps/api/src/app/workflows-v2/mappers/notification-template-mapper.ts (hunk 1 above)
Comment (rifont) [reply apps/api/src/app/workflows-v2/mappers/notification-template-mapper.ts]: Please disregard. I read `dashboard` in the PR title, it should be `feat(dashboard, api)` to signify changes in both.

File: apps/dashboard/src/components/workflow-status.tsx [hunk 2]
```diff
@@ -32,7 +40,7 @@ export const WorkflowStatus = (props: WorkflowStatusProps) => {
   const text = statusRenderData[status].text;
 
   return (
-    <Badge variant={badgeVariant}>
+    <Badge variant={badgeVariant} className="border-none py-1">
```
Comment (desiprisg) [inline apps/dashboard/src/components/workflow-status.tsx]: Why the extra styles here?

File: apps/api/src/app/workflows-v2/mappers/notification-template-mapper.ts [hunk 3]
```diff
@@ -63,10 +63,22 @@ function toMinifiedWorkflowDto(template: NotificationTemplateEntity): WorkflowLi
     updatedAt: template.updatedAt || 'Missing Updated At',
     stepTypeOverviews: template.steps.map(buildStepTypeOverview).filter((stepTypeEnum) => !!stepTypeEnum),
     createdAt: template.createdAt || 'Missing Create At',
-    status: WorkflowStatusEnum.ACTIVE,
+    status: getWorkflowStatus(template),
```
Comment (djabarovgeorge) [inline apps/api/src/app/workflows-v2/mappers/notification-template-mapper.ts]: i think this one should work as the workflow should be updated.

```suggestion
    status: template.status,
```

File: apps/dashboard/src/components/workflow-status.tsx (hunk 2 above)
Comment (BiswaViraj) [reply apps/dashboard/src/components/workflow-status.tsx]: after the rebase it doesnt needs the py-1, removing it. but it needs the "border-none" for the inactive state

**Pull request merged**